mesonwrap/tools/reviewtool.py

```diff
@@ -157,15 +159,21 @@ def check_url(self, upwrap):
                      self._branch in upwrap.source_url,
                      fatal=self.options.strict_version_in_url)
 
+    def is_permitted_file(self, filename):
+        if filename in self.permitted_files:
+            return True
+        if filename.endswith('.h.meson'):
+            return True
+        return False
```

Comment:
The PR heading doesn't seem to match the code?

Reply:
I guess it should say "configuration header templates". This came about when there was a wrap that needed a conf header with a custom name rather than just plain `config.h`.